cmd/cosign/cli/verify/verify.go

```diff
@@ -48,22 +48,27 @@ import (
 // nolint
```

Comment:
Also need to update:
* https://github.com/sigstore/cosign/blob/f2c360eb97e52fa7766ecde370f1a48b910d7404/cmd/cosign/cli/dockerfile.go
* https://github.com/sigstore/cosign/blob/f2c360eb97e52fa7766ecde370f1a48b910d7404/cmd/cosign/cli/manifest.go
* https://github.com/sigstore/cosign/blob/03e66aad02cf7c987ea489cdeda1fd580b6b1fc6/cmd/cosign/cli/verify/verify_attestation.go
* https://github.com/sigstore/cosign/blob/03e66aad02cf7c987ea489cdeda1fd580b6b1fc6/cmd/cosign/cli/verify/verify_blob.go

And also update where `o.CertVerify.X` is added to the command, like line 180 in `cmd/cosign/cli/verify.go`

Reply:
thank you, I missed those parts.

Reply:
thanks for updating these!